File: src/com/ketanolab/simidic/SplashActivity.java
package com.ketanolab.simidic;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.animation.Animation;
import android.view.animation.Animation.AnimationListener;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;

public class SplashActivity extends Activity {

	private ImageView imagenKetanolab;

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_splash);

		imagenKetanolab = (ImageView) findViewById(R.id.imagenKetanolab);

		Animation animation = AnimationUtils.loadAnimation(this, R.anim.anim_splash);

		animation.setAnimationListener(new AnimationListener() {

			public void onAnimationStart(Animation animation) {

			}

			public void onAnimationRepeat(Animation animation) {

			}

			public void onAnimationEnd(Animation animation) {
				Intent intent = new Intent(SplashActivity.this, MainActivity.class);
				startActivity(intent);
				SplashActivity.this.finish();
			}
		});

		imagenKetanolab.startAnimation(animation);
	}
}

File: app/src/main/java/com/ketanolab/simidic/SplashActivity.kt
package com.ketanolab.simidic

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.animation.Animation
import android.view.animation.Animation.AnimationListener
import android.view.animation.AnimationUtils
import android.widget.ImageView

class SplashActivity : Activity() {
    private var imagenKetanolab: ImageView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        imagenKetanolab = findViewById<View>(R.id.imagenKetanolab) as ImageView
        val animation = AnimationUtils.loadAnimation(this, R.anim.anim_splash)
        animation.setAnimationListener(object : AnimationListener {
            override fun onAnimationStart(animation: Animation) {}
            override fun onAnimationRepeat(animation: Animation) {}
            override fun onAnimationEnd(animation: Animation) {
                val intent = Intent(this@SplashActivity, MainActivity::class.java)
                startActivity(intent)
                finish()
            }
        })
        imagenKetanolab!!.startAnimation(animation)
    }
}